### PR TITLE
chore: add "@auto-it/released"

### PR DESCRIPTION
**Pull request by Yakutoc**
Для того что у нас было так  

Screenshot 2023-06-29 at 14 12 58

### Документация 

[Released Plugin](https://github.com/intuit/auto/tree/main/plugins/released)

----------

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.580.5409097893.swift)  
[color_asdk_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.580.5409097893.kt)  
[color_asdk_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.580.5409097893.ts)  
[color_asdk_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_xml--canary.580.5409097893.xml)  
[color_plasma_b2c_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.580.5409097893.swift)  
[color_plasma_b2c_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.580.5409097893.kt)  
[color_plasma_b2c_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.580.5409097893.ts)  
[color_plasma_b2c_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.580.5409097893.xml)  
[color_sberHealth_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.580.5409097893.swift)  
[color_sberHealth_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.580.5409097893.kt)  
[color_sberHealth_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.580.5409097893.ts)  
[color_sberHealth_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.580.5409097893.xml)  
[color_sbermarket_business_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.580.5409097893.swift)  
[color_sbermarket_business_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.580.5409097893.kt)  
[color_sbermarket_business_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.580.5409097893.ts)  
[color_sbermarket_business_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.580.5409097893.xml)  
[color_sbermarket_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.580.5409097893.swift)  
[color_sbermarket_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.580.5409097893.kt)  
[color_sbermarket_metro_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.580.5409097893.swift)  
[color_sbermarket_metro_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.580.5409097893.kt)  
[color_sbermarket_metro_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.580.5409097893.ts)  
[color_sbermarket_metro_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.580.5409097893.xml)  
[color_sbermarket_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.580.5409097893.ts)  
[color_sbermarket_selgros_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.580.5409097893.swift)  
[color_sbermarket_selgros_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.580.5409097893.kt)  
[color_sbermarket_selgros_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.580.5409097893.ts)  
[color_sbermarket_selgros_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.580.5409097893.xml)  
[color_sbermarket_wlbusiness_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.580.5409097893.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.580.5409097893.kt)  
[color_sbermarket_wlbusiness_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.580.5409097893.ts)  
[color_sbermarket_wlbusiness_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.580.5409097893.xml)  
[color_sbermarket_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.580.5409097893.xml)  
[color_sberonline_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.580.5409097893.swift)  
[color_sberonline_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.580.5409097893.kt)  
[color_sberonline_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.580.5409097893.ts)  
[color_sberonline_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.580.5409097893.xml)  
[color_sberprime_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.580.5409097893.swift)  
[color_sberprime_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.580.5409097893.kt)  
[color_sberprime_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.580.5409097893.ts)  
[color_sberprime_xml--canary.580.5409097893.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.580.5409097893.xml)  
[shadow_sbermarket_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.580.5409097893.ts)  
[typo_mage_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.580.5409097893.swift)  
[typo_mage_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.580.5409097893.kt)  
[typo_mage_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.580.5409097893.ts)  
[typo_plasma_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.580.5409097893.swift)  
[typo_plasma_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.580.5409097893.kt)  
[typo_plasma_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.580.5409097893.ts)  
[typo_ruler_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.580.5409097893.swift)  
[typo_ruler_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.580.5409097893.kt)  
[typo_ruler_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.580.5409097893.ts)  
[typo_sage_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.580.5409097893.swift)  
[typo_sage_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.580.5409097893.kt)  
[typo_sage_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.580.5409097893.ts)  
[typo_sbermarket_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.580.5409097893.swift)  
[typo_sbermarket_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.580.5409097893.kt)  
[typo_sbermarket_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.580.5409097893.ts)  
[typo_soulmate_ios-swift--canary.580.5409097893.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.580.5409097893.swift)  
[typo_soulmate_kotlin--canary.580.5409097893.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.580.5409097893.kt)  
[typo_soulmate_react-native--canary.580.5409097893.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.580.5409097893.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.217.1-canary.580.5409097893.0
  npm install @salutejs/cra-template-plasma-basic-template@2.3.1-canary.580.5409097893.0
  npm install @salutejs/plasma-colors@0.7.1-canary.580.5409097893.0
  npm install @salutejs/plasma-core@1.122.1-canary.580.5409097893.0
  npm install @salutejs/plasma-hope@1.217.1-canary.580.5409097893.0
  npm install @salutejs/plasma-icons@1.151.1-canary.580.5409097893.0
  npm install @salutejs/cra-template-plasma-shop-template@2.2.3-canary.580.5409097893.0
  npm install @salutejs/plasma-temple@1.166.1-canary.580.5409097893.0
  npm install @salutejs/plasma-tokens-b2b@1.22.1-canary.580.5409097893.0
  npm install @salutejs/plasma-tokens-b2c@0.31.1-canary.580.5409097893.0
  npm install @salutejs/plasma-tokens-core@0.2.1-canary.580.5409097893.0
  npm install @salutejs/plasma-tokens-web@1.37.1-canary.580.5409097893.0
  npm install @salutejs/plasma-tokens@1.56.1-canary.580.5409097893.0
  npm install @salutejs/plasma-typo@0.34.1-canary.580.5409097893.0
  npm install @salutejs/plasma-ui@1.198.1-canary.580.5409097893.0
  npm install @salutejs/plasma-web@1.217.1-canary.580.5409097893.0
  npm install @salutejs/plasma-cy-utils@0.63.1-canary.580.5409097893.0
  npm install @salutejs/plasma-sb-utils@0.120.1-canary.580.5409097893.0
  npm install @salutejs/plasma-tokens-utils@0.27.1-canary.580.5409097893.0
  # or 
  yarn add @salutejs/plasma-b2c@1.217.1-canary.580.5409097893.0
  yarn add @salutejs/cra-template-plasma-basic-template@2.3.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-colors@0.7.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-core@1.122.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-hope@1.217.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-icons@1.151.1-canary.580.5409097893.0
  yarn add @salutejs/cra-template-plasma-shop-template@2.2.3-canary.580.5409097893.0
  yarn add @salutejs/plasma-temple@1.166.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-tokens-b2b@1.22.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-tokens-b2c@0.31.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-tokens-core@0.2.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-tokens-web@1.37.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-tokens@1.56.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-typo@0.34.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-ui@1.198.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-web@1.217.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-cy-utils@0.63.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-sb-utils@0.120.1-canary.580.5409097893.0
  yarn add @salutejs/plasma-tokens-utils@0.27.1-canary.580.5409097893.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-580/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-580/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-580/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-580/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-580/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-580/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-580/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-580/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-580/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-580/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-580/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-580/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-580/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-580/temple-storybook/

File: package.json [hunk 1]
```diff
@@ -137,7 +138,14 @@
         }
       ],
       "conventional-commits",
-      "./auto-plugins/dist/update-deps.js"
+      "./auto-plugins/dist/update-deps.js",
```
Comment (Yeti-or) [inline package.json:141]: надо будет это отсюда тоже убрать
потом

**Review by Yeti-or — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Yakutoc) [reply package.json:141]: @Yeti-or Объясни, почему ну или кинь в меня докой )

**Review by kayman233 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Yeti-or) [reply package.json:141]: а потому что я не уверен что оно вообще работает

Comment (Yeti-or) [reply package.json:141]: например оно просит переменную

https://github.com/salute-developers/plasma/blob/dev/auto-plugins/src/update-deps.ts#L10

```ts

const baseUrl = process.env.UPDATE_SERVICE_BASE_URL;
export default class UpdateDepsPlugin implements IPlugin {
...
```

а она даже не задана в репозитории

**Pull request merged**